Neos.Flow/Classes/I18n/EelHelper/TranslationHelper.php

```diff
@@ -135,10 +135,11 @@ protected function translateByShortHandString($shortHandString)
         $shortHandStringParts = explode(':', $shortHandString);
         if (count($shortHandStringParts) === 3) {
             list($package, $source, $id) = $shortHandStringParts;
-            return $this->createTranslationParameterToken($id)
+            $translation = $this->createTranslationParameterToken($id)
                 ->package($package)
                 ->source(str_replace('.', '/', $source))
                 ->translate();
+            return empty($translation) ? $id : $translation;
```

Comment:
```suggestion
            return $translation ?? $id;
```

Reply:
empty is not equal to `??` null coalescing, as it will also capture empty strings:
https://3v4l.org/bnFlZ
I guess an empty string can be considered a missing translation?